integration_test/workflow_tests/test_packages/ingest/test_multuple_granules_happy_path.py

```diff
@@ -0,0 +1,197 @@
+import json
+import logging
+import time
+import uuid
+from unittest import TestCase
+
+import boto3
+
+import helpers
+
+
+class TestMultipleGranulesHappyPath(TestCase):
+    def test_multiple_granules_happy_path(self):
+        """
+        If multiple granules are provided, should store them in DB as well as in recovery bucket.
+        """
+        try:
+            # Set up Orca API resources
+            # ---
+            my_session = helpers.create_session()
+            granule_id_1 = uuid.uuid4().__str__()
+            granule_id_2 = uuid.uuid4().__str__()
+            provider_id = uuid.uuid4().__str__()
+            provider_name = uuid.uuid4().__str__()
+
+            collection_name = uuid.uuid4().__str__()
+            collection_version = uuid.uuid4().__str__()
+            bucket_name = "orca-sandbox-s3-provider"  # standard bucket where initial file exists
```

Comment:
In an ideal world all files from this bucket should be generated at test-time to avoid collisions. At minimum, this should be a static string.

Reply:
existing test assumes the files are already present in the bucket.

we could create another task for this like mentioned above for randomizing file names.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-628